app/graphql/types/enum/installment_tag.rb

```diff
@@ -0,0 +1,8 @@
+class Types::Enum::InstallmentTag < Types::Enum::Base
+  value 'MAIN_STORY', '', value: :main_story
+  value 'SIDE_STORY', '', value: :side_story
+  value 'SPINOFF', '', value: :spinoff
+  value 'MAIN_ENTRY', '', value: :main_entry
```

Comment:
Is there a difference between MAIN_STORY and MAIN_ENTRY?

Reply:
hmm, good catch, me do bad copy pasta.